### DISCUSSION TITLE
connecting email to webhook to 3commas

### DISCUSSION THREAD

**Discussion by bbaarrttoo** [soranoo/TradingView-Free-Webhook-Alerts#26] [General]
I've been trying to connect my TV alerts to my 3commas account bout the 3commas bot is unable to get the webhooks. while I was trying to find the error I decided to come back to the post and noticed that there was a new version(ngrok). My doubt is if there is any way i can extract only a certain part of the mail so that way maybe the 3commas bot wont fail. If you can give me any advice that migth be useful I will be really thankful.
P.S. I live in a place where English is not the main language, if you need me to re explain something there is no problem. Also I am really new to programing and your progam has saved me a lot of time and money(which I hope i do not have to spend if there is no solution to the problem). sorry this is taking so long but if you know of any other apps that allow you to create a trading bot that you know that the email to webhook progam function with can you send me the name.
thanks in advance

**Comment by soranoo** [comment]
Did you receive an email in HTML format?
If yes, I recommend switching it to plain text. Simply turn off the `Send email` and turn on `Send email-to-SMS`.
![image](https://user-images.githubusercontent.com/46896789/210132892-83feec04-34e5-4592-a2a3-58ec44c87eb0.png)
